rasa/shared/utils/io.py

```diff
@@ -333,7 +331,7 @@ def env_var_constructor(loader: BaseConstructor, node: ScalarNode) -> Text:
 replace_environment_variables()
 
 
-def read_yaml(content: Text, reader_type: Union[Text, List[Text]] = "safe") -> Any:
+def read_yaml(content: Text, reader_type: Union[Text, List[Text]] = "rt") -> Any:
```

Comment:
Excellent catch 🎉

Reply:
I actually had a second thought - since the default reader_type must have been set to `safe` for a reason, let's set this to `rt` when being used inside the `validate_assistant_id_in_config` function, wdyt?

Reply:
@ancalita Yeah, I had the same thought as well.
But It's not being used directly in that function. It's been used in  the [read_validated_yaml](https://github.com/RasaHQ/rasa/blob/main/rasa/shared/utils/io.py#L573) method which is also being called by many.

I can put the change here instead.

Reply:
I've limited the use of `rt` to only the `validate_assistant_id_in_config` method.